cloudstack/resource_cloudstack_network_offering.go

```diff
@@ -50,6 +50,85 @@ func resourceCloudStackNetworkOffering() *schema.Resource {
 				Type:     schema.TypeString,
 				Required: true,
 			},
+			"domain_id": {
+				Type: schema.TypeList,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+				Optional:    true,
+				Description: "the ID of the containing domain(s), null for public offerings",
+			},
+			"network_rate": {
```

Comment:
Need to add ForceNew:    true, for all these options to ensure new network offering is created. As of now, if I create an offering and then change the network_rate, I see the plugin crashing.

Reply:
done